docs/stencil-docs/customizing-checkout/checkout-sdk-quickstart.md

```diff
@@ -37,171 +34,28 @@ For example, `cd cornerstone`
-    account_recentitems: getAccount,
-    account_downloaditem: getAccount,
-    editaccount: getAccount,
-    account_inbox: getAccount,
-    account_saved_return: getAccount,
-    account_returns: getAccount,
-    account_paymentmethods: getAccount,
-    account_addpaymentmethod: getAccount,
-    account_editpaymentmethod: getAccount,
-    login: getLogin,
-    createaccount_thanks: getLogin,
-    createaccount: getLogin,
-    getnewpassword: getLogin,
-    forgotpassword: getLogin,
-    blog: noop,
-    blog_post: noop,
-    brand: () => import('./theme/brand'),
-    brands: noop,
-    cart: () => import('./theme/cart'),
-    category: () => import('./theme/category'),
-		checkout: () => import('./theme/checkout'),
-    compare: () => import('./theme/compare'),
-    page_contact_form: () => import('./theme/contact-us'),
-    error: noop,
-    404: noop,
-    giftcertificates: () => import('./theme/gift-certificate'),
-    giftcertificates_balance: () => import('./theme/gift-certificate'),
-    giftcertificates_redeem: () => import('./theme/gift-certificate'),
-    default: noop,
-    page: noop,
-    product: () => import('./theme/product'),
-    amp_product_options: () => import('./theme/product'),
-    search: () => import('./theme/search'),
-    rss: noop,
-    sitemap: noop,
-    newsletter_subscribe: noop,
-    wishlist: () => import('./theme/wishlist'),
-    wishlists: () => import('./theme/wishlist'),
-};
-...
-```
-
-3. Save the file.
-
-## Preparing the <span class="fn">checkout.html</span> template file
-
-1. Open the <span class="fn">checkout.html</span> file in your text editor.
-2. Comment out the following statement:
-
- `{{{ checkout.checkout_content }}}`
-
-`<!--{{{ checkout.checkout_content }}}-->`
-
-3. On the next line, add the following:
-
-<div class="HubBlock-header">
-    <div class="HubBlock-header-title flex items-center">
-        <div class="HubBlock-header-name">checkout.html</div>
-    </div><div class="HubBlock-header-subtitle"></div>
-</div>
-
-<!--
-title: "checkout.html"
-subtitle: ""
-lineNumbers: true
--->
-
-```html
-<script>window.__webpack_public_path__ = "{{cdn 'assets/dist/'}}";</script>
-<script src="{{cdn 'assets/dist/theme-bundle.main.js'}}"></script>
-
-<script>
-    window.stencilBootstrap("{{page_type}}", {{jsContext}}).load();
-</script>
-```
-
-4. Save the file.
+## Obtaining the JavaScript loader file
+1. Navigate to the [Open source checkout](https://github.com/bigcommerce/checkout-js) repository. 
+2. Clone the respository.
+3. Move into the checkout-js directory.
+5. Run `npm ci` to install dependencies.
+6. Build out your custom checkout page by making changes to the source code located in the checkout.js folder. 
+7. Run `npm run build` to build the source code.
+
+To view your changes, run `npm run dev:server` and enter the local URL for autoloader-js in **Custom Checkout Settings** in the **Script URL** field.
 
 ## Logging the Checkout Object

```

Comment:
I believe this is no longer relevant, as we have removed the line that produces the logging
https://github.com/bigcommerce/dev-docs/pull/687/files#diff-b2be35fdaa92fb65df49f14c43a34df2L69